plenum/test/bls/test_sign_validation_for_key_proof_exist.py

```diff
@@ -0,0 +1,95 @@
+import pytest
+
+from plenum.common.exceptions import PoolLedgerTimeoutException
+from plenum.common.keygen_utils import init_bls_keys
+from plenum.common.util import hexToFriendly
+from plenum.test.bls.helper import check_bls_key
+from plenum.test.helper import sdk_send_random_and_check
+from plenum.test.node_catchup.helper import waitNodeDataEquality
+from plenum.test.node_request.helper import sdk_ensure_pool_functional
+from plenum.test.pool_transactions.helper import sdk_send_update_node, sdk_pool_refresh
+
+nodes_wth_bls = 0
+
+
+def test_switched_off_sign_validation_for_key_proof_exist(looper,
+                                                          txnPoolNodeSet,
+                                                          sdk_pool_handle,
+                                                          sdk_wallet_stewards,
+                                                          sdk_wallet_client,
+                                                          monkeypatch):
+    '''
+    Add BLS key without BLS key proof. Test that when ALIDATE_SIGN_WITHOUT_BLS_KEY_PROOF = True,
+    node use key sent without proof.
+    '''
+    for n in txnPoolNodeSet:
+        monkeypatch.setattr(n.poolManager.reqHandler, 'doStaticValidation', lambda req: True)
+        n.bls_bft.bls_key_register._pool_manager.config.VALIDATE_SIGN_WITHOUT_BLS_KEY_PROOF = True
```

Comment:
Since config is a singleton, it needs to be changed in a fixture with yield

Reply:
Added fixture for VALIDATE_SIGN_WITHOUT_BLS_KEY_PROOF but without yeld. Is it ok?